### ISSUE TITLE
Mod list

### ISSUE THREAD

**Issue by l-2-j** [neos-modding-group/world#6]
It'd be rad to display our verified mod list in game also

**Comment by rampa3** [comment]
Definitely. Some style of interactive catalogue, with options to open weblinks/cloud spawn latest DLL files from GitHub repos maybe?

**Comment by l-2-j** [comment]
> Some style of interactive catalogue, with options to open weblinks

Yes

> spawn latest DLL files

That'd be neat, but; How would you do hash verification with that? Since that's like the most important part if you are actually handling the files

**Comment by rampa3** [comment]
> > spawn latest DLL files
> 
> That'd be neat, but; How would you do sha256 verification with that? Since that's like the most important part if you actually handling the files

How? An automated proxy repository, that pulls down only verified latest versions of the DLLs from manifest into stock for cloud spawn.

**Comment by l-2-j** [comment]
> An automated proxy repository, that pulls down only verified latest versions of the DLLs from manifest into stock for cloud spawn.

That'd require more GHA work, so that depend on https://github.com/neos-modding-group/neos-mod-manifest/pull/102
